benchmarks/CMakeLists.txt

```diff
@@ -1,11 +1,10 @@
 set (BENCHMARK_DATA_DIR "${CMAKE_CURRENT_SOURCE_DIR}/realdata/") # does this ever get used?
 
 add_c_benchmark(create_benchmark)
-add_c_benchmark(real_bitmaps_benchmark)
+# We exclude POSIX tests from Visual Studio default build
+if(NOT WIN32)
```

Comment:
Future work is to check for a POSIX runtime rather than use this coarse switch. But it doesn't seem to be doable from CMake. Excluding this one test isn't a big deal in the case of an odd Win32 POSIX-compatible system.